### PR TITLE
spider: Promote to Release status

### PR DESCRIPTION
**Pull request by kingthorin**
Updated changelog and build file.

As discussed via email.

Related to zaproxy/zaproxy#3113

Signed-off-by: kingthorin <kingthorin@users.noreply.github.com>

### CODE REVIEW

File: addOns/openapi/CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ### Changed
 - Maintenance changes.
+- Promoted to Release status, with the migration of the core Spider (3113).
```
Comment (thc202) [inline addOns/openapi/CHANGELOG.md:12]: Was added to other add-on.

Comment (kingthorin) [reply addOns/openapi/CHANGELOG.md:12]: Oh crud, duh!

I must have had it open while commenting on that other PR :cry:

**Comment by kingthorin** [comment]
Tweaked

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Review by ricekot — APPROVED** [review]

**Pull request merged**